media/css/products/relay/faq.scss

```diff
@@ -0,0 +1,33 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/protocol/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+.c-relay-faq-wrapper {
+    background-color: #09204d;
+
+    .column-wrapper {
```

Comment:
This should drop to one column on smaller pages. Also, you might be able to do it with CSS columns and lose the extra wrappers. They're pretty finicky though so if you try/tried and it didn't work let me know.